app/src/main/kotlin/com/wire/android/services/PersistentWebSocketService.kt

```diff
@@ -135,7 +140,11 @@ class PersistentWebSocketService : Service() {
     }
 
     private fun generateForegroundNotification() {
-        val notification: Notification = Notification.Builder(this, WEB_SOCKET_CHANNEL_ID)
+        if (notificationChannelsManager.shouldCreateChannel(WEB_SOCKET_CHANNEL_ID)) {
```

Comment:
the whole  fun `notificationChannelsManager.shouldCreateChannel` should be removed, Android checks if channel already exist by itself.
I mentioned that in Oussamas PR, but it was merged faster than I accepted my review :)

Reply:
Cool, right - I got a little confused because I couldn't find it on develop 😅 
Ok, so I removed it from RC as well 😄